mockup1/assets/filtering.js

```diff
@@ -56,10 +56,14 @@
         selectors.push('[data-version~="' + filters.version.value + '"]');
     }
     qsOn = SELECTOR_PREFIX + selectors.join('');
-    if (selectors.length > 0) {
+    if (0 === selectors.length) {
```

Comment:
if `selectors` is empty, this will be the default view, e.g. generated by the backend (so it also works w/o JS enabled).
The backend should be responsible for hiding these specifications.

Reply:
It'd be right when you load the page without filters from the server initially, yes. But if you filter client-side, then reset filters client-side, `selectors.length` will be `0` again, and your two new lines will not be executed.

Reply:
Good point! That's indeed a valid use case.